src/components/NewsletterForm/NewsletterFormResearchDropDown.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import NewsletterFormItem from './NewsletterFormItem';
+
+type NewsletterFormResearchDropDownProps = {
+  handleChange?: (event: React.FormEvent<HTMLSelectElement>) => void;
+  value: string;
+};
+
+export const NewsletterFormResearchDropDown = ({
+  handleChange,
+  value
+}: NewsletterFormResearchDropDownProps) => {
+  return (
+    <NewsletterFormItem type="dropdown">
+      <label
+        className="newsletter-form__item-label newsletter-form__item-label--dropdown"
+        htmlFor="dropdown"
+      >
+        Which of these best describes you?
+      </label>
+      <select
+        id="edit-research-dd"
```

Comment:
`-options` makes more sense than `-dd` as a suffix. dd is also coupling the type to a particular form element which could potentially change